packages/storybook-ui-components/stories/Avatar.stories.tsx

```diff
@@ -1,47 +1,47 @@
-import React, { FunctionComponent, ReactElement } from "react";
-import { withKnobs, text, boolean, select } from "@storybook/addon-knobs";
-import { action } from "@storybook/addon-actions";
-import { Avatar, AvatarProps, Icon } from "@cockroachlabs/ui-components";
-import { StoryContainer, StoryDescription } from "../layout";
+import React, { FunctionComponent, ReactElement } from 'react';
```

Comment:
Why does your formatter change all the quotes in this PR?

Reply:
prettier had it change it automatically to single quotes, updated the changes.

Reply:
Gotcha. We use `eslint` to apply `prettier` along with other linting rules. Make sure your formatter is configured that way. If you just "run prettier" on the code you won't get the correct result unfortunately.

Reply:
Done